client-apis/decrypt.md

```diff
@@ -124,11 +124,11 @@ The AAD used in decryption is the [Message Body AAD](#message-body-aad.md), cons
   If the message contains non framed data, then this value is 1.
 - Content Length: TODO
 
-If [max body length](#max-body-length) is specified on input and the input message contains a
-[message body frame](#message-body.md#framed-data) with an encrypted content length greater than this value,
-Decrypt MUST not perform the decryption algorithm on that message body frame and MUST fail.
-If [max body length](#max-body-length) is specified on input and the input message is unframed and has a
-[encrypted content length](#message-body.md#encrypted-content-length) greater than this value,
+If [max body length](#max-body-length) is specified on input and the input message is [framed data](../data-format/message-body.md#framed-data)
```

Comment:
Is this `>` or `>=`?

https://github.com/aws/aws-encryption-sdk-python/blob/00c7e1dbfd0aade7ccb61e5e5735e26a8a239c38/src/aws_encryption_sdk/streaming_client.py#L770
vs
https://github.com/aws/aws-encryption-sdk-javascript/blob/master/modules/decrypt-node/src/verify_stream.ts#L109

cc @mattsb42-aws